website/docusaurus.config.js

```diff
@@ -347,41 +347,47 @@ const config = {
             title: 'Documentation',
             items: [
               {
-                label: 'Installing Podman Desktop',
+                label: 'Installing',
                 to: '/docs/installation',
               },
               {
                 label: 'Onboarding for containers',
                 to: '/docs/onboarding/containers',
               },
-              {
-                label: 'Onboarding for Kubernetes',
-                to: '/docs/onboarding-for-kubernetes',
-              },
               {
                 label: 'Working with containers',
                 to: '/docs/working-with-containers',
               },
               {
-                label: 'Migrating from Docker',
-                to: '/docs/migrating-from-docker',
+                label: 'Working with Kubernetes',
+                to: '/docs/kubernetes',
               },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
               {
-                label: 'Working with Compose',
-                to: '/docs/compose',
+                label: 'Discord',
+                href: 'https://discord.com/channels/852634929845239818/1085121643782873118',
               },
+
               {
-                label: 'Working with Kubernetes',
-                to: '/docs/kubernetes',
+                label: 'IRC',
+                href: 'https://web.libera.chat/#podman-desktop',
+              },
+              {
+                label: 'Matrix',
```

Comment:
As a new user, I'd be unsure what Matrix is. Is it the movie? 

Would be good adding: Chat to the front.